persistance/src/main/java/cz/muni/fi/pv254/entity/Game.java

```diff
@@ -3,11 +3,15 @@
 import javax.persistence.*;
 import javax.validation.constraints.NotNull;
 import javax.validation.constraints.Pattern;
+import java.util.HashSet;
+import java.util.Set;
 
 /**
  *
  * @author Šimon Baláž
  */
+@Table(name= "games")
+@Entity
 public class Game {
     
     @Id

```

Comment:
We used to add here:
@GeneratedValue(strategy = GenerationType.IDENTITY)
not sure if needed